providers/go-feature-flag/pkg/provider.go

```diff
@@ -33,6 +34,7 @@ type Provider struct {
 	cache                  gcache.Cache
 	cacheTTL               time.Duration
 	cacheDisable           bool
+	cacheMetricsDisable    bool
```

Comment:
I would change the name to `dataCollectorDisable`, I did a wrong choice naming that cache in the beginning.

```suggestion
	dataCollectorDisable    bool
```

Reply:
Right :) if you change name it changes bit of other code too. Commited:
```
	if options.DisableCache || options.DisableDataCollector {
		return nil
	}
```